pkgs/development/compilers/ghc/8.6.5-binary.nix

```diff
@@ -73,6 +74,10 @@ stdenv.mkDerivation rec {
       url = "${downloadsUrl}/${version}/ghc-${version}-x86_64-apple-darwin.tar.xz";
       sha256 = "0s9188vhhgf23q3rjarwhbr524z6h2qga5xaaa2pma03sfqvvhfz";
     };
+    powerpc64le-linux = {
+      url = "https://downloads.haskell.org/~ghc/8.6.5/ghc-8.6.5-powerpc64le-fedora29-linux.tar.xz";
```

Comment:
Version numbers and the domain should probably be substituted like above.

Reply:
Fixed and squashed.